content/en/blog/_posts/2019-07-09-some-apis-are-being-deprecated.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: 'APIs Deprecating Soon: Here\'s What You Need to Know'
+date: 2019-07-08
+---
```

Comment:
It might also help to have a `slug: api-deprecations-in-1-16` entry here as well